hw/drivers/sensors/icp101xx/src/icp101xx.c

```diff
@@ -352,13 +359,15 @@ read_otp(struct sensor_itf *itf, struct icp101xx_cfg *cfg, int16_t *calibration_
     data_write[2] = crc;
```

Comment:
This is strange that no length is specify for the CRC calculation. Buffer Size is 10 Bytes. Data use 2 Bytes.
Buffer is not fully initialize. I don't understand how it works

Reply:
This is are coded to 2 bytes. This is why this not possible to write more than one register in a single access.

Reply:
The CRC is always computed on 2 bytes and written in the 3rd byte. Only 3 bytes are evaluated in the cmpute_crc() API. But you're rigth it would be better to give the CRC size as parameter to be more clear. I will update it.

Reply:
Same problem for Read access. This is we cannot more than 2 bytes a single access.